frontend/src/routes/report/InputRow.tsx

```diff
@@ -23,7 +24,7 @@ interface InputRowProps {
  * @param {any} props.form - The form object containing the values.
  * @returns {JSX.Element} - The rendered input row component.
  */
-const InputRow = ({ setRows, form }: InputRowProps) => {
+const InputRow = ({ setRows, form }: InputRowProps): JSX.Element => {
```

Comment:
Explicitly returning Jsx.Element seems a bit redundant, it's quite clear that InputRow is a component. I'm thinking maybe it should be its own file?